src/components/BottomNavigation/BottomNavigation.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from "next/image";
+import { Avatar, AvatarFallback, AvatarImage } from "@radix-ui/react-avatar";
+import Link from "next/link";
+import MoreIcon from "@/public/icons/more.svg";
+import HomeIcon from "@/public/icons/home.svg";
+import FavouriteRecipesIcon from "@/public/icons/favourite-recipes.svg";
+
+const BottomNavigation = () => {
+  return (
+    <>
+      <div className="flex fixed bottom-3 left-3">
```

Comment:
maybe wrap these divs with parent element that is only `fixed` to bottom and position elements inside using flex? that way would be more scalable and responsive